### ISSUE TITLE
[Idea] Allow some special characters

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/vscode-allint#6]
Hi,
at the moment i receive often the warning for use of special characters.
I would love it to allow some special characters like `_`. At the moment this character is used for the hungarian notation, but in the future I would like to add my Affix to procedures in table or page extensions.

**Comment by StefanMaron** [comment]
Yeah, I think I will add the definition of special signs to the settings.
